### PR TITLE
Various problems

### PR DESCRIPTION
**Pull request by dolphingarlic**
Just some problems from OIs (many have official editorials, but I didn't link them yet)

### CODE REVIEW

**Comment by jessechoe10** [comment]
Fix the failed checks...

**Review by bqi343 — CHANGES_REQUESTED** [review]

File: content/4_Gold/MST.mdx [hunk 1]
```diff
@@ -53,6 +53,33 @@ export const problems = {
...
+			'2016 - Rollercoaster Railroad',
+			'https://oj.uz/problem/view/IOI16_railroad',
+			'Very Hard',
+			true,
+			['MST'],
+			''
+		),
+		new Problem(
+			'CEOI',
+			'2008 - Information',
```
[the remark this answers is not in the document]
Comment (dolphingarlic) [reply content/4_Gold/MST.mdx:76]: I think the official editorial has one: https://www.ceoi2008.de/files/short-spoiler.pdf

I never actually tried this problem yet

File: content/4_Gold/MST.mdx [hunk 2]
```diff
@@ -53,6 +53,33 @@ export const problems = {
...
+		new Problem(
+			'APIO',
+			'2013 - Toll',
+			'https://oj.uz/problem/view/APIO13_toll',
+			'Very Hard',
+			false,
+			['MST', 'Bitmask'],
+			''
+		),
+		new Problem(
```
[the remark this answers is not in the document]
Comment (dolphingarlic) [reply content/4_Gold/MST.mdx:65]: Should I just mark them as "Insane" then? I think they're great problems for practice and there isn't really anywhere else to put them

**Review by bqi343 — COMMENTED** [review]
> I think the official editorial has one:

https://codeforces.com/blog/entry/68131

Doesn't seem like anyone has a simple implementation?

> I think they're great problems for practice

I disagree ... I think adding the IOI 2016 problem, in particular, is sort of a spoiler (and isn't really useful to anyone)? I also don't think directed MST is relevant to undirected MST.

**Comment by dolphingarlic** [comment]
@bqi343 I removed IOI Railroad and CEOI Information, but kept APIO Toll and just marked it as "Insane"

**Comment by dolphingarlic** [comment]
Also, should I display the radial sweep problems or should I leave that for another PR?

**Comment by bqi343** [comment]
Display them.

**Comment by dolphingarlic** [comment]
@bqi343 is it good to merge? I can't re-request a review from you for some reason

**Review by bqi343 — APPROVED** [review]

**Pull request merged**